src/psyclone/nemo.py

```diff
@@ -352,6 +353,14 @@ def view(self, indent=0):
         '''
         print(self.indent(indent) + self.coloured_text + "[]")
 
+    def reference_accesses(self, var_accesses):
+        '''Get all variable access information. It calls the corresponding
+        kernel schedule function.
+        :param var_accesses: \
```

Comment:
Blank line before `:param:` please and please document the purpose of `var_accesses` as wel as its type.